ls1intum/Artemis
# [Feature] Add common buttons of non-programming exercise in detail views

### Checklist

### Motivation and Context


The detail views of text, file upload, and modeling exercises do not provide buttons to the most-used views like in the programming exercise.

### Description

This PR adds the buttons "Edit", "Delete", "Participations", "Submissions", "Scores", and "Export Submissions" button into the detail views of text, file upload, and modeling exercises.

### Steps for Testing


1. Log in to Artemis
2. Navigate to Course Administration
3. Create a text, file upload, and modeling exercises (if they don't exist)
4. Navigate to the detail views of each exercise and make sure the new buttons are there and work as expected

### Test Coverage



non-programming-exercise-detail-common-actions.component.ts 92.31%

### Screenshots


Buttons for modeling exercises:

![image](https://user-images.githubusercontent.com/11298674/112275866-5d108b80-8c80-11eb-868a-d261432ffc52.png)

Buttons for text exercises:

![image](https://user-images.githubusercontent.com/11298674/112275939-6f8ac500-8c80-11eb-91be-51f5eaa5e10c.png)

Buttons for file upload exercises:

![image](https://user-images.githubusercontent.com/11298674/112276230-cbede480-8c80-11eb-8b77-ea36088646c0.png)
